chat-plugins/wifi.js

```diff
@@ -377,30 +388,33 @@ var commands = {
 		switch (target) {
 		case 'staff':
 			if (!this.can('warn', null, room)) return;
-			reply = '<strong>Staff commands:</strong><br />' +
-			        '- question or qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway (Requires: % @ # & ~)<br />' +
-			        '- lottery or lg <em>User, Prize[, Number of Winners]</em> - Starts a lottery giveaway (Requires: % @ # & ~)<br />' +
-			        '- changequestion - Changes the question of a question giveaway (Requires: giveaway host)<br />' +
-			        '- changeanswer - Changes the answer of a question giveaway (Requires: giveaway host)<br />' +
-					'- viewanswer - Shows the answer in a question giveaway (only to giveaway host/giver)<br />' +
-			        '- end - Forcibly ends the current giveaway (Requires: % @ # & ~)<br />';
+			reply =
+				'<strong>Staff commands:</strong><br />' +
+				'- question or qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway (Requires: % @ # & ~)<br />' +
+				'- lottery or lg <em>User, Prize[, Number of Winners]</em> - Starts a lottery giveaway (Requires: % @ # & ~)<br />' +
+				'- changequestion - Changes the question of a question giveaway (Requires: giveaway host)<br />' +
+				'- changeanswer - Changes the answer of a question giveaway (Requires: giveaway host)<br />' +
+				'- viewanswer - Shows the answer in a question giveaway (only to giveaway host/giver)<br />' +
+				'- end - Forcibly ends the current giveaway (Requires: % @ # & ~)<br />';
 			break;
 		case 'game':
 		case 'giveaway':
 		case 'user':
 			if (!this.canBroadcast()) return;
-			reply = '<strong>Giveaway participation commands: </strong> (start with /giveaway, except for /ga) <br />' +
-			        '- guess or /ga <em>answer</em> - Guesses the answer for a question giveaway<br />' +
-			        '- viewanswer - Shows the answer in a question giveaway (only to host/giver)<br />' +
-			        '- remind - Shows the details of the current giveaway (can be broadcast)<br />' +
-			        '- join or joinlottery - Joins a lottery giveaway<br />' +
-			        '- leave or leavelottery - Leaves a lottery giveaway<br />';
+			reply =
```

Comment:
I usually don't code-review plugin files because they're too big and they're self-contained anyway, so they're usually not to PS conventions that we don't configure into `jscs`.